include/nbl/asset/utils/CJITIncludeLoader.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _NBL_ASSET_C_JIT_INCLUDE_LOADER_H_INCLUDED_
+#define _NBL_ASSET_C_JIT_INCLUDE_LOADER_H_INCLUDED_
+
+#include <string>
+#include <unordered_map>
+
+class CJITIncludeLoader
```

Comment:
in order to be able to call `make_smart_refctd_ptr<CJITIncludeLoader>` the class needs to inherit from `IReferenceCounted`

Reply:
btw it needs to inherit from `IShaderCompiler::IIncludeLoader `